fhir-resource-tooling/src/main/java/com/ibm/cohort/tooling/fhir/ValueSetImporter.java

```diff
@@ -138,8 +168,29 @@ static void runWithArgs(String[] args, PrintStream out) throws IOException {
 		}
 	}
 
+	private static void putToCos(ValueSetImporterArguments arguments, FhirContext fhirContext, ValueSet vs, String vsFileName, AmazonS3 cosClient) {
+		ObjectMetadata metadata = new ObjectMetadata();
+		byte[] arr = fhirContext.newJsonParser().encodeResourceToString(vs).getBytes();
+		metadata.setContentLength(arr.length);
+		PutObjectRequest put = new PutObjectRequest(arguments.bucket, vsFileName, new ByteArrayInputStream(arr), metadata);
+		cosClient.putObject(put);
+	}
+
+	public AmazonS3 createClient(String api_key, String service_instance_id, String endpoint_url, String location)
+	{
+		SDKGlobalConfiguration.IAM_ENDPOINT = "https://iam.cloud.ibm.com/identity/token";
```

Comment:
Should we be hardcoding this? Seems like this would be tying us to IBM cloud and what if this endpoint changed?